docs/UserGuide.adoc

```diff
@@ -167,15 +180,9 @@ diagnosis and prescription given to the patient
 to patient with NRIC S7534436H, along with their index which can be used to display
 actual consultation details
 
-=== Patient Management - view past patient records: [coming in v1.2]
-
-View the past consultation records of each patient +
 
-=== Patient Management - generate invoice: [coming in v2.0]
-
-Generates the invoice for a given prescription. After selecting the patient, select the consultation to view in detail and then select the desired prescription.
-
```

Comment:
I think all of us need to grant more space between the feature sections when editing the user guide to enhance readability. A larger space is needed between different modules.